scripts/check-requirements.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/sh
+
+STAGED_REQ_FILES=0
+
+for FILE in "$@"
+do
+    if [ "$FILE" = "requirements.in" ]; then
+        STAGED_REQ_FILES=$((STAGED_REQ_FILES+1))
+    elif [ "$FILE" = "requirements.txt" ]; then
+        STAGED_REQ_FILES=$((STAGED_REQ_FILES+1))
+    fi
+done
+
+if [ "$STAGED_REQ_FILES" -eq 1 ]; then
+    printf "\e[37;41mChanges to requirements.in should be compiled to requirements.txt by running 'make freeze-requirements'. The file requirements.txt file should not be edited manually.\e[0m\n"
```

Comment:
The warning message probably has one more "file" than required, very minor point, just doesn't read correctly.

Reply:
No problem, let me see if I can reword it